sdk.yaml

```diff
@@ -14,148 +19,84 @@ Agent Identifier:
       Includes a product component indicating the type of runtime, as discrete from the OS type.
       This does not always include a version (e.g. for `browser` runtime emitted by `ably-js`
       and the `uwp` runtime emitted by `ably-dotnet`).
-Client Instance Creation:
-  .specification: RSC1
+Authentication:
+  .documentation:
+    - https://ably.com/docs/core-features/authentication
+  .specification: [AO1, AO2, RSC17, RSA7, RSA15, RTN2d, TO3a]
   .synopsis: |
-    An SDK announcing support for any of these construction inputs must support those
-    inputs for all client types it offers.
-  With Client Options:
-  With String:
-    .specification: RSC1a
-    .synopsis: |
-      Where that string is treated as an API key if it includes a ':' (colon) delimiter,
-      otherwise it is treated as a token string.
-Options:
-  .synopsis: |
-    `ClientOptions` that apply to both client types (REST and Realtime).
-  Agents:
-    .specification: RSC7d6
+    Mechanisms authenticating with the Ably service.
+    Encapsulated by the `AuthOptions` class.
+    Implies the existence of the `clientId` client option, the identifier of the client represented by the client instance.
+  Basic:
+    .documentation:
+      - https://ably.com/docs/core-features/authentication#basic-authentication
+    .specification: [RSA11, RSC1, TO3j1, AO2a]
     .synopsis: |
-      `agents`:
-      For use only by other Ably-authored SDKs, on a need-to-have basis.
-  Authentication:
+      Capability to use a 'raw', full Ably key string, as obtained from the dashboard, for authentication.
```

Comment:
A few points about this sentence:

- I don't think it needs to be classified as "raw"
- I think it should be named "API key" rather than "full Ably key string"
- It might be obtained from the Control API as well as the dashboard (I don't think where it comes from is relevant here though)

Reply:
https://github.com/ably/features/pull/41/commits/a5f4bab1481ffc4e90cbb652235ee37600cae49f

Reply:
Resolving this conversation on @lmars' behalf, as I've addressed the comment and wish to close the pull request cleanly.